tests/integration/java-integration/run_local_build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ueo pipefail
+
+SCRIPT_ROOT="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+cd "$SCRIPT_ROOT"
+
+CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-docker}
+"$CONTAINER_RUNTIME" run \
+ -i \
```

Comment:
Should this be `-t` rather than `-i` for CI usage without any interaction?

Reply:
fixed